src/elements/Image/ImageGroup.jsx

```diff
@@ -0,0 +1,21 @@
+import { classes, getElementType } from '../../lib';
+import { Enum } from '../../lib/PropTypes';
+
+export default {
+  name: 'SuiImageGroup',
+  props: {
+    size: Enum(['mini', 'tiny', 'small', 'medium', 'large', 'big', 'huge', 'massive']),
```

Comment:
You can use Enum.Size here: https://github.com/Semantic-UI-Vue/Semantic-UI-Vue/blob/master/src/lib/PropTypes.js#L30

Reply:
Ah right, will fix in a moment